### DISCUSSION TITLE
Welcome to Echelon Discussions!

### DISCUSSION THREAD

**Discussion by aubymori** [echelon-theme/echelon#43] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Didrick81** [comment]
Hi, new member from Norway here. So I've downloaded Echelon, along with Firefox 115 ESR and I'm sorry if some of this comes across as dumb as I'm quite new to this. I believe I've followed the instructions to the best of my abilities but right now I'm stuck at where it tells me to  extract the zip of the theme/the contents of the profile folder to the root directory of the profile folder. I keep looking in the profile folder to see if I have overlooked a zip file but I can't seem to find any there. There is still the Echelon zip folder in my downloads though, do I still need that for something? Sorry again if I sound like a complete noob but that's because I am haha but I love the whole Aero aesthetic and would love to rock Firefox with the blurred glass Aero bar on my Windows 11. I do already have the DWMBlurGlass installed which makes my other windows look almost just like things did back in the day of Vista and W7 and learning that Echelon is compatible with DWMBlurGlass, I decided to give it a shot. But now I'm stuck at the copy contents folder of the my profile folder with zip. Any help? Thanks in advance!

**Reply by aubymori** [reply]
You need to extract the zip of Echelon and put the contents of the Profile Folder folder into your Firefox profile folder.

**Comment by halfmexican** [comment]
Hi! I daily drive Linux. I only use windows for VR, but I adore this project since I've riced my desktop. It's incredibly pleasing to the eye and I hope it development continues!

**Comment by denny466** [comment]
Omg im so glad i found this. Thank you so much for making Echelon! <3

How tricky is it to adapt onto the next ESR version? 115 ESR will cease receiving security patches in March 2025 unfortunately: https://support.mozilla.org/en-US/kb/firefox-users-windows-7-8-and-81-moving-extended-support
